ecosystem/lava-sdk/src/lavasession/consumerSessionManager.ts

```diff
@@ -132,7 +133,8 @@ export class ConsumerSessionManager {
       // For LAVA's initialization, we need to allow the pairing to be updated twice
       // This condition permits the pairing to be overwritten just once for the same epoch
       // After this one-time allowance, any attempt to overwrite will result in an error
-      if (epoch != 0) {
+      // Also allow fetch pairing for the same epoch in case of emergency mode(virtualEpoch != 0)
+      if (epoch != 0 && virtualEpoch == 0) {
```

Comment:
we should not be resetting the state on virtual epochs, or touching the pairing, just increase the allowed CU when a request is performed by multiplying same as in the rpcconsumer